File: Eiler_project_Python/EL7.py

import math

def is_prime(num):
	n = round(math.sqrt(num))
	for x in range(2, n + 1):
		if num % x == 0:
			return False
	return True

prime_nums = []
j = 2
while len(prime_nums) < 10001:
	if is_prime(j):
		prime_nums.append(j)
	j += 1
print(prime_nums[-1])

File: Eiler_project_JavaScript/EL7.js
function isSimple(num){
	let sqr_num = Math.round(num ** (1/2))
	for (let div = 2; div <= sqr_num; div++){
		if (num % div == 0) return false
	}
	return true
}

let arrayOfSimple = []

let i = 1
while (arrayOfSimple.length <= 10001){
	if (isSimple(i)) { arrayOfSimple.push(i) }
	i++
}

console.log(arrayOfSimple[10001])